batavia/types/Bytes.js

```diff
@@ -326,30 +317,13 @@ Bytes.prototype.__add__ = function(other) {
                 "can't concat " + type_name(other) + ' to bytes'
             )
         }
-    } else {
-        throw new exceptions.NotImplementedError.$pyclass('Bytes.__add__ has not been implemented')
+    } else {        
+        throw new exceptions.TypeError.$pyclass("can't concat bytes to " + type_name(other))
     }
 }
 
 Bytes.prototype.__sub__ = function(other) {
```

Comment:
At column 13: [(no-trailing-spaces) Trailing spaces not allowed.](http://eslint.org/docs/rules/no-trailing-spaces)